app/icon_cache_file.go

```diff
@@ -0,0 +1,59 @@
+package app
+
+import (
+	"os"
+	"path/filepath"
+	"sync"
+
+	"fyne.io/fyne/v2"
+)
+
+var once sync.Once
+
+func (a *fyneApp) CachedIconPath() string {
```

Comment:
I don't think this needs to be changed to a public function...

Reply:
Done. Even I can't say why I changed it (?)